stocks2.lua

```diff
@@ -0,0 +1,544 @@
+                    text_hpen = {fg = COLOR_BLACK, bg = COLOR_GREY},
+                    cursor_pen = {fg = COLOR_LIGHTGREEN, bg = COLOR_DARKGREY},
+                }),
+            },
+        }),
+        widgets.Panel({
+            view_id = "right_panel",
+            frame = {t = 0, b = 0, r = 0, l = 72},
+            frame_style = gui.INTERIOR_FRAME,
+            subviews = {
+                widgets.Label({
+                    text = "Filters (Alt+Key toggles)",
+                    frame = {t = 0, l = 0},
+                    view_id = "filters_header_label",
+                }),
+                widgets.Label({
+                    text = "to be filled in...",
+                    frame = {t = 1, l = 0},
+                    view_id = "filters_label",
+                }),
+                widgets.Label({
+                    text = {
+                        {text = "Shift-C", pen = COLOR_LIGHTGREEN}, {text = ": Clear All", pen = COLOR_WHITE}, NEWLINE,
+                        {text = "Shift-E", pen = COLOR_LIGHTGREEN}, {text = ": Enable All", pen = COLOR_WHITE}, NEWLINE,
+                        {text = "Shift-G", pen = COLOR_LIGHTGREEN}, {text = ": Toggle Grouping", pen = COLOR_WHITE}
+                    },
+                    frame = {t = 8, l = 0},
+                    view_id = "filters_label2",
+                }),
+                widgets.Label({
+                    text = {
+                        {text = "-+", pen = COLOR_LIGHTGREEN}, {text = ": Min Qual: ", pen = COLOR_WHITE}, {text = "tbd", pen = COLOR_BROWN}, NEWLINE,
+                        {text = "/*", pen = COLOR_LIGHTGREEN}, {text = ": Max Qual: ", pen = COLOR_WHITE}, {text = "tbd", pen = COLOR_BROWN}, NEWLINE,
+                        {text = "Shift-W", pen = COLOR_LIGHTGREEN}, {text = ": Min Wear: ", pen = COLOR_WHITE}, {text = "tbd", pen = COLOR_BROWN}
+                    },
+                    frame = {t = 12, l = 0},
+                    view_id = "filters_qual_wear",
+                }),
+                widgets.Label({
+                    text = {
+                        {text = "Actions (", pen = COLOR_BROWN}, {text = "...", pen = COLOR_LIGHTGREEN}, {text = " Items)", pen = COLOR_BROWN}
+                    },
+                    frame = {t = 16, l = 0},
+                    view_id = "actions_item_count_label",
+                }),
+                widgets.Label({
+                    text = {
+                        {text = "Shift-Z", pen = COLOR_LIGHTGREEN}, {text = ": Zoom    ", pen = COLOR_WHITE},
+                        {text = "-D", pen = COLOR_LIGHTGREEN}, {text = ": Dump", pen = COLOR_WHITE}, NEWLINE,
+                        {text = "Shift-F", pen = COLOR_LIGHTGREEN}, {text = ": Forbid  ", pen = COLOR_WHITE},
+                        {text = "-M", pen = COLOR_LIGHTGREEN}, {text = ": Melt", pen = COLOR_WHITE}, NEWLINE,
+                        {text = "Shift-T", pen = COLOR_LIGHTGREEN}, {text = ": Mark for Trade", pen = COLOR_GREY},
```

Comment:
These should all be `HotkeyLabel`s. The "Shift" portion of the key is not shown, it's just printed as a capital letter, so that will save you some space.